scripts/docker-compose-openfn.yml

```diff
@@ -26,25 +26,27 @@ services:
   web:
     platform: linux/x86_64/v8
     image: 'openfn/lightning:latest'
```

Comment:
I'll release a new lightning version with the adaptor cache enabled (`v2.10.7` or greater)

Reply:
(Done.)